src/lib/datepicker/month-view.ts

```diff
@@ -147,9 +173,79 @@ export class MatMonthView<D> implements AfterContentInit {
       this.selectedChange.emit(selectedDate);
     }
 
+    this._userSelected();
+  }
+
+  _userSelected(): void {
```

Comment:
nit: I don't think this needs to be its own function, `this._userSelected()` is only a couple characters shorter than `this._userSelection.emit()`